.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
### Your Readme File :memo:
![readme](https://user-images.githubusercontent.com/93039359/187618567-ff055dbd-503f-4631-b1ec-de6cdaba938e.png)

- [ ] You have added a readme file :blush: but did not add your information in the readme file such as **Project name**, **Project description**, **Getting Started commands**, your **social account handles**. Please add them to make your readme file descriptive because it is the documentation of your project so please take it seriously :thinking: